### ISSUE TITLE
Fluire. При вводе в поле дня числа с 1 по 9, выдаётся ни о чём не говорящая ошибка “Incorrect input body”

### ISSUE THREAD

**Issue by KolobokMysnoy** [VK-BMSTU-QA/homework-1-autumn-2023#47]
Приоритет: высокий

![image](https://github.com/yarikTri/homework-1-autumn-2023/assets/71096930/6cda37d2-283a-4128-b2b9-8459dd732c23)

URL: http://84.23.54.19/profile

Сценарий воспроизведения:
  1) Авторизоваться
  2) Открыть профиль
  3) Ввести день с 1 до 9, год с 1970 и любой месяц
  4) Нажать на "Save changes"

Ожидаемый результат: появление надписи "Successfully changed user info"
Фактический результат: появляется ошибка "incorrect input body"

**Comment by stepanpopov** [comment]
Приоритет можно понизить, так как баг не затрагивает основной функционал приложения.

**Comment by KolobokMysnoy** [comment]
> Приоритет можно понизить, так как баг не затрагивает основной функционал приложения.

Не согласен. Мне кажется, что это является основным функционалом, так как это задача профиля в смене данных, а он не может её выполнить

**Comment by yarikTri** [comment]
Кажется, в воспроизведении упустил ввод месяца и года

**Comment by KolobokMysnoy** [comment]
> Кажется, в воспроизведении упустил ввод месяца и года

Исправил.

**Comment by yarikTri** [comment]
Аппрув

**Comment by stepanpopov** [comment]
аппрув

**Comment by GodofRussia** [comment]
аппрув

**Comment by mistergahan** [comment]
аппрув
